### ISSUE TITLE
Feature Request: Applescript control for sliders

### ISSUE THREAD

**Issue by ihorner** [kyleneideck/BackgroundMusic#278]
I'd like to be able to set sound source volumes via AppleScript or similar — I can't seem to find any information regarding this, and the dictionary seems to only apply to setting output.

I'm using BetterTouchTool to send volume levels from a midi device with faders, and I'd like to associate the faders with different sources in BackgroundMusic.

**Comment by kyleneideck** [comment]
Good idea. As far as I can tell, the AppleScript API only supports reading and changing the output device so far.

**Comment by broady** [comment]
@kyleneideck how can you do that? that'd be pretty useful on its own.

sorry, I'm a bit of an AppleScript noob. is there some way to explore the API of an app?
